packs/chef/actions/lib/shellhelpers.py

```diff
@@ -43,17 +42,18 @@ def __init__(self, parser_options):
             self.parser.add_argument(k, lg, **kwargs)
             self._keyname[lg.lstrip('-')] = k
 
+
     def parse(self, argv=None):
-        argv = argv or sys.argv
```

Comment:
I made this change because I prefer to be explicit - `parse_args` does indeed use `sys.arg`v if you don't explicitly provide `args` argument, but it's more explicit and clear if you explicitly pass it to the method.